ibis/backends/pyspark/tests/test_basic.py

```diff
@@ -162,6 +162,19 @@ def test_filter(client, filter_fn, expected_fn):
     tm.assert_frame_equal(result.toPandas(), expected.toPandas())
 
 
+def test_filter_with_window_op(client):
```

Comment:
Can you add this test to the backend tests? Maybe close to https://github.com/ibis-project/ibis/blob/master/ibis/backends/tests/test_generic.py#L132

Reply:
In general, shared backend tests are the default place for these things unless there is a good reason to not to.

Reply:
Makes sense, moved the test to backends/tests/test_generic.py